sys/include/stdio_base.h

```diff
@@ -8,10 +8,7 @@
  */
 
 /**
- * @defgroup    sys_stdio STDIO abstraction
- * @ingroup     sys
- *
- * @brief       Simple standard input/output (STDIO) abstraction for RIOT
+ * @ingroup     sys_stdio STDIO abstraction
```

Comment:
```suggestion
 * @addtogroup  sys_stdio STDIO abstraction
```

I think this is this priority issue thing again. Doxygen does not include the content of the header file into the generated page either.